src/utils/media-utils.js

```diff
@@ -362,6 +362,7 @@ export const traverseMeshesAndAddShapes = (function() {
         o.name !== "Floor_Plan" &&
         o.name !== "Ground_Plane"
       ) {
+        o.updateMatrices();
```

Comment:
Hmm, whats this about? This should only get called if you are about to read an objects matrix. Guessing that maybe happens in ammo-shape? If so would be good to do this there instead. (we are trying to keep upadteMatricies/matrixWorldNeedsUpdate as close as possible to the code that makes them needed so we can spot places where we are breaking the contract by mistake)